custom_components/octopus_energy/climate.py

```diff
@@ -56,6 +56,21 @@ async def async_setup_default_sensors(hass, config, async_add_entities):
     ),
     "async_boost_heat_pump_zone"
   )
+  platform.async_register_entity_service(
```

Comment:
question: Is this exposed in the app? Can it cause issues if tweaked?

Reply:
It isn't, but I expect most users using this integration will be setting this via the local API anyway. Can remove if you'd rather not expose functionality that's not exposed via the app for now.

Reply:
If it's something a user would want to automate, let's keep it in. We can put a warning in the docs if tweaking it can cause issues.